modin/experimental/cloud/rayscale.py

```diff
@@ -148,7 +148,10 @@ def _conda_requirements(self):
 
         reqs.extend(self._get_python_version())
 
-        reqs.append(self._get_modin_version())
+        if not any(map(lambda pkg: pkg.startswith("modin"), self.add_conda_packages)):
```

Comment:
Note that this would break for something like `modinfoo`. I suggest using `re.match(r'modin(\W|$)', pkg)` which looks for exact string `modin` followed up by a NON-alphanumeric character OR end of string

Reply:
Good suggestion!